src/job_cache/job_cache.cpp

```diff
@@ -16,11 +16,10 @@
  */
 
 // Open Group Base Specifications Issue 7
+#include "types.h"
```

Comment:
nit: This should be below 

```c++
#define _XOPEN_SOURCE 700
#define _POSIX_C_SOURCE 200809L
```

no?

Reply:
For some reason my editor keeps using the wrong clang-format invocation that does this but re-doing clang-format doesn't fix this, its annoying.